src/app/components/layouts/expandable-text/expandable-text.component.ts

```diff
@@ -0,0 +1,84 @@
+import {
+    AfterViewInit,
+    Component,
+    EventEmitter,
+    Input,
+    Output,
+} from '@angular/core';
+import { TranslationService } from 'angular-l10n';
+import { isNullOrUndefined } from 'util';
+
+
+let nextId:number = 0;
+
+@Component({
+    selector: 'tc-expandable-text',
+    styles:   [require('./expandable-text.component.scss')],
+    template: require('./expandable-text.component.html')
+})
+export class ExpandableTextComponent implements AfterViewInit
+{
+    @Input()
+    public collapsed:boolean = true;
```

Comment:
naming entsprechend der Component
```suggestion
    public expanded:boolean = false;
```